.github/workflows/test.yml

```diff
@@ -1,20 +1,22 @@
-name: "Test typescript-action"
-on:
-  pull_request:
-  push:
-    branches:
-      - master
-      - 'releases/*'
+name: "Test setup-nuget-exe"
+
+on: [pull_request, push]
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        nuget-version: [latest, preview]
+        os: [ubuntu-latest, macOS-latest, windows-latest]
     steps:
     - uses: actions/checkout@v1
 
     - run: npm ci
     - run: npm run build
     - run: npm test
     - uses: ./
-      with: 
-        milliseconds: 1000
+      with:
+        nuget-version: ${{ matrix.nuget-version }}
+        nuget-api-key: 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a
```

Comment:
I think it would be good to have this test to use something like this `nuget-api-key: ${{ secrets.NuGetAPIKey }}` (same as Readme file), so that users follow that pattern, even when we are not executing workflows for this repository.